pkg/service/replay/service.go

```diff
@@ -33,7 +33,7 @@ type Service interface {
 	Instrument(ctx context.Context) (*InstrumentState, error)
 	GetNextTestRunID(ctx context.Context) (string, error)
 	GetAllTestSetIDs(ctx context.Context) ([]string, error)
-	RunTestSet(ctx context.Context, testSetID string, testRunID string, appID uint64, serveTest bool) (models.TestSetStatus, error)
+	RunTestSet(ctx context.Context, testSetID string, testRunID string, appID uint64, serveTest bool, runApp bool) (models.TestSetStatus, error)
```

Comment:
We should make it pass via a test run config as there are more than 3 arguments now.